test-apps/display-test-app/android/imodeljs-test-app/app/build.gradle

```diff
@@ -20,12 +22,19 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    buildFeatures {
+        viewBinding true
+    }
 }
 
 dependencies {
-    implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'com.google.android.material:material:1.0.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    implementation 'com.bentley.itwin:iTwinAndroidLibrary-release:1.0@aar'
+    implementation 'androidx.appcompat:appcompat:1.5.1'
+    implementation 'com.google.android.material:material:1.6.1'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
+    implementation 'androidx.navigation:navigation-ui:2.5.2'
+    implementation 'com.github.itwin:mobile-native-android:3.5.6'
```

Comment:
@aruniverse This version needs to be updated when we adopt a new add-on release. I believe there's a checklist somewhere that will need to be updated to include this file but I don't know where it is.

Reply:
Done via this script now, https://github.com/iTwin/itwinjs-core/blob/master/common/scripts/update-imodeljs-native-version.sh

Reply:
Add to documentation [here](https://dev.azure.com/bentleycs/iModelTechnologies/_wiki/wikis/iModelTechnologies.wiki/2036/Creating-a-new-imodel.js-addon) and [here](https://dev.azure.com/bentleycs/iModelTechnologies/_wiki/wikis/iModelTechnologies.wiki/12161/Creating-a-patch-addon).

Reply:
Script updated: https://github.com/iTwin/itwinjs-core/pull/4594